src/routes/docs/quick-starts/deno/+page.markdoc

```diff
@@ -19,200 +21,337 @@ If this is your first time using Appwrite, create an account and create your fir
 ![Create project screen](/images/docs/quick-starts/create-project.png)
 {% /only_light %}
 
-Then, under **Integrate with your server**, add an **API Key** with the following scopes.
+Next, click **Functions** > **Create Function**, and select **Deno** as the runtime.
```

Comment:
Wait these server quick starts aren't supposed to be coupled with Appwrite Functions 🧐